src/pageLayout/components/RenamablePageTitle.tsx

```diff
@@ -45,8 +45,6 @@ const RenamablePageTitle: FC<Props> = ({
     if (onClickOutside) {
       onClickOutside(e)
     }
-
-    setEditingState(false)
```

Comment:
what happens if `onClickOutside` is undefined?

Reply:
good call. we needed to move the `setEditingState` call into the cleanup function in the `useEffect` hook.